bookwyrm/settings.py

```diff
@@ -368,6 +368,9 @@
 BASE_URL = f"{PROTOCOL}://{NETLOC}"
 CSRF_TRUSTED_ORIGINS = [BASE_URL]
 
+if DEBUG:
+    CORS_ORIGIN_WHITELIST = [BASE_URL]
```

Comment:
Hello!

I searched the Django docs for this setting, but could only find it in the docs for the [django-cors-headers](https://pypi.org/project/django-cors-headers/#cors-allowed-origins-sequence-str) package. Is this setting useful without installing that?

If you can, I'd like to understand how to reproduce the CSRF failure that this fixes. I can run a non-grok setup myself with:

```
DOMAIN="localhost:1335"
DEBUG="true"
USE_HTTPS="no"
```

and login works, and could not see any breakage.

Thanks!!